website/docs/r/project_ip_whitelist.html.markdown

```diff
@@ -18,34 +18,58 @@ When you remove an entry from the whitelist, existing connections from the remov
 
 ## Example Usage
 
+### Using Cidr Block
 ```hcl
 resource "mongodbatlas_project_ip_whitelist" "test" {
-    project_id = <PROJECT-ID>
-
-    whitelist {
-      cidr_block = "1.2.3.4/32"
-      comment    = "cidr block for tf acc testing"
-    }
-    whitelist {
-      ip_address = "2.3.4.5"
-      comment    = "ip address for tf acc testing"
-    }
-    whitelist {
-      cidr_block = "3.4.5.6/32"
-      comment    = "cidr block for tf acc testing"
-    }
-    whitelist {
-      ip_address = "4.5.6.7"
-      comment    = "ip address for tf acc testing"
-    }
- }
+  project_id = "<PROJECT-ID>"
+  cidr_block = "1.2.3.4/32"
+  comment    = "cidr block for tf acc testing"
+}
+```
+
+### Using IP Address
+```hcl
+resource "mongodbatlas_project_ip_whitelist" "test" {
+  project_id = "<PROJECT-ID>"
+  ip_address = "2.3.4.5"
+  comment    = "ip address for tf acc testing"
+}
+```
+
+### Using AWS Security Group
+```hcl
+resource "mongodbatlas_network_container" "test" {
```

Comment:
Add comment above this 
// In order to use AWS Security Group(s) VPC Peering must be enabled.